jdaviz/configs/default/plugins/export/export.py

```diff
@@ -474,11 +468,16 @@ def export(self, filename=None, show_dialog=None, overwrite=False,
                 # in case one was never created in the parent plugin
                 self.plugin_plot_selected_widget = f'IPY_MODEL_{plot.model_id}'
 
+            if self.overwrite_warn and not overwrite:
+                if raise_error_for_overwrite:
+                    raise FileExistsError(f"{filename} exists but overwrite=False")

```

Comment:
Just in case. I think exposing actual var value is more useful than hardcoding what we think it is.
```suggestion
                    raise FileExistsError(f"{filename} exists but overwrite={overwrite}")
```

Reply:
Good call, committed.